### PR TITLE
Improve travis CI and fix go test issue

### PR DESCRIPTION
**Pull request by chenrui333**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,27 +1,27 @@
...
 before_script:
   - pushd ..; git clone https://github.com/containerd/project; popd
 
 script:
   - DCO_VERBOSITY=-q ../project/script/validate/dco
   - ../project/script/validate/fileheader ../project/
-  - go test -race -covermode=atomic -c .
-  - sudo ./zfs.test -test.root -test.v -test.coverprofile=coverage.txt
+  - GOGC=75 golangci-lint run
+  - go test -v -race -covermode=atomic -coverprofile=coverage.txt ./...
```
Comment (AkihiroSuda) [inline .travis.yml]: needs sudo

Comment (AkihiroSuda) [reply .travis.yml]: same for AUFS: https://github.com/containerd/aufs/pull/18/files#r363200701

Comment (chenrui333) [reply .travis.yml]: Got you.

**Comment by chenrui333** [comment]
Will debug the aufs build first and then come back on this build.

**Comment by chenrui333** [comment]
@AkihiroSuda should be good now, I limited the scope of this PR of just improving the travis CI.

File: .travis.yml [hunk 2]
```diff
@@ -1,27 +1,21 @@
...
 
 before_script:
   - pushd ..; git clone https://github.com/containerd/project; popd
 
 script:
   - DCO_VERBOSITY=-q ../project/script/validate/dco
   - ../project/script/validate/fileheader ../project/
-  - go test -race -covermode=atomic -c .
-  - sudo ./zfs.test -test.root -test.v -test.coverprofile=coverage.txt
+  - sudo $GOROOT/bin/go test -test.root -test.v -race -covermode=atomic -coverprofile=coverage.txt ./...
```
Comment (AkihiroSuda) [inline .travis.yml]: What is the advantage of this change?

Comment (AkihiroSuda) [reply .travis.yml]: ping @chenrui333, please consider reverting `go test` part

**Comment by estesp** [comment]
Sorry that we never finalized this PR--didn't realize it was still open. We are migrating to GitHub actions in PR #34 so any changes to Travis are no longer applicable. Thanks for your contribution.

**Pull request closed without merging**